gui-themes/themes/zeit-amp/scripts/js/plugins/button-pagination.js

```diff
@@ -5,28 +5,21 @@ define([
     'plugins/pagination',
     'dust',
     'lib/utils',
-    'lib/helpers/display-toggle',
     'tmpl!themeBase/plugins/after-button-pagination',
     'tmpl!themeBase/plugins/before-button-pagination'
-], function(Backbone, plugins, paginationPlugin, dust, utils, displayToggle) {
+], function(Backbone, plugins, paginationPlugin, dust, utils) {
     delete plugins.pagination;
     plugins['button-pagination'] = function(config) {
         paginationPlugin(config);
 
         utils.dispatcher.once('add-all.posts-view', function(view) {
-            var current = view.collection.length,
-                data = {};
+            var data = {};
 
             dust.renderThemed('themeBase/plugins/before-button-pagination', data, function(err, out) {
-                    var  el = Backbone.$(out);
-                    displayToggle(el, false);
-                    view.$el.prepend(el);
+                view.$el.prepend(Backbone.$(out));
```

Comment:
It should be `view.$el.before` here, but I did not get this to work.

The final output must be
```
<amp-live-list data-gimme="posts.view">
    <div update on="tap:zon-live-list.update"></div>
    <ul items data-gimme="posts.list">
        …
    </ul>
    <div pagination></div>
</amp-live-list>
```